content/applications/hr/recruitment/new_job.rst

```diff
@@ -143,68 +173,274 @@ Sections are indicated by a gray background, while questions have a white backgr
+
+
+   .. image:: new_job/layout.png
+      :align: center
+      :alt: The layout field when properly configured for a conditional question.
+
+Live Sessions
+^^^^^^^^^^^^^
+
+If the question is for a live session, activate the :guilabel:`Question Time Limit` option. A
+:guilabel:`seconds` field appears. Enter the amount of time allotted for the applicant to enter the
+answer, in seconds.
+
+Options
+~~~~~~~
+
+Next, configure the various options for the interview form. Click the :guilabel:`Options` tab to
+view all the options to configure, by category.
 
 Questions
-~~~~~~~~~
+*********
 
-- :guilabel:`Layout`: Choose how the questions should be displayed.
+- :guilabel:`Pagination`: Choose how the questions should be displayed.
 
-  - :guilabel:`One page with all the questions`: Display all sections and questions at the same
-    time.
+  - :guilabel:`One page per question`: Display a single question on each page.
   - :guilabel:`One page per section`: Display each section with the corresponding questions on an
     individual page.
-  - :guilabel:`One page per question`: Display a single question on each page.
+  - :guilabel:`One page with all the questions`: Display all sections and questions at the same
+    time.
 
-- :guilabel:`Progression Mode`: Display the progress the candidate is making, either as a
-  :guilabel:`Percentage`, or as a :guilabel:`Number`.
-- :guilabel:`Survey Time Limit`: Check this box to limit the time allowed to complete the form. When
-  selected, a field to enter the minutes appears next to the checked box. Enter the time (using an
-  XX:XX minute/second format) in the field.
-- :guilabel:`Selection`: Display the entire form (e.g. :guilabel:`All questions`), or only a random
-  selection of questions from each section (:guilabel:`Randomized per section`).
 
-  .. note::
-     Although it is a selectable option, the :guilabel:`Randomized per section` option is not
-     typically selected for an interview form.
+- :guilabel:`Display Progress as`: Choose how to display the percentage of questions remaining to
+  answer. This option only appears if either :guilabel:`One page per question` or :guilabel:`One
+  page per section` is selected for :guilabel:`Pagination`.
 
-Scoring
-~~~~~~~
+  - :guilabel:`Percentage left`: Display the remaining amount in a percentage (%).
+  - :guilabel:`Number`: Display the remaining amount in a numerical value.
+
+
+- :guilabel:`Question Selection`: Choose which questions are presented.
+
+  - :guilabel:`All questions`: Display the entire form, with all questions form all sections.
+  - :guilabel:`Randomized per Section`: Display only a random selection of questions from each
+    section.
+
+
+- :guilabel:`Back Button`: Activate this option if the applicant is able to click a back button to
+  go back to previous questions.
+
+.. note::
+   Although it is a selectable option, the :guilabel:`Randomized per section` option is not
+   typically selected for an interview form.
```

Comment:
I mentioned this on the review for 15, but instead of saying "not typically selected" I would emphasize that this shouldn't be selected unless they are ok with receiving surveys that are incomplete, as it means not all of the included questions will be presented to applicants. (Instead of just saying don't do it, say why)

Reply:
This is npw changed to how I changed it in 15